libethereum/State.h

```diff
@@ -98,6 +98,40 @@ DEV_SIMPLE_EXCEPTION(IncorrectAccountStartNonceInState);
 
 class SealEngineFace;
 
+
+
+namespace detail
+{
+struct Change
+{
+	enum Kind: int
+	{
+		balance,
```

Comment:
Changed to uppercase, but I would like to keep typed enum, because this avoids another namespace nesting. In the end, it is a class internal to State API.